crates/client-api/src/auth.rs

```diff
@@ -58,6 +59,9 @@ impl SpacetimeCreds {
     pub fn decode_token(&self, public_key: &DecodingKey) -> Result<SpacetimeIdentityClaims, JwtError> {
         decode_token(public_key, self.token()).map(|x| x.claims)
     }
+    fn from_signed_token(token: String) -> Self {
+        Self { token }
+    }
     /// Mint a new credentials JWT for an identity.
     pub fn encode_token(private_key: &EncodingKey, identity: Identity) -> Result<Self, JwtError> {
```

Comment:
Are we unable to remove this function yet?

Reply:
Not yet, because we are using it in some internal functions. We will need to remove it when we change how we are accepting tokens, but it doesn't need to be removed for this PR.